cypress/integration/e2e/table/range-selection.spec.ts

```diff
@@ -216,7 +128,15 @@ context("Shift range selection", () => {
       cy.getWorkerShift(test.startShiftCell).trigger("dragstart");
       cy.getWorkerShift(test.endShiftCell).trigger("drop");
       cy.useAutocomplete(test.desiredShiftCode);
-      validateChange(test);
+      cy.get('p[data-cy*="cell"]')
```

Comment:
Proponuję robić snapshot tylko tego wiersza który zmieniamy: (myślę, że można pominąć testy na kolumnach, ponieważ wybór kolumn - to nie jest funkcjonalność na której nam zależy). Motywacja jest taka:
1. test będzie przechodzić szybciej
2. będziemy w stanie upewnić się że dane zapisane w pliku snapshot są takie których my oczekujemy. Ja teraz próbowałem sprawdzić i w takim formacie jak to jest tera mi się nie udało)

Reply:
1.będzie szybszy, ale nie wydaje mi się że znacząco, można to zmienić żeby sprawdzić
2.to jest duży minus, natomiast wynika z bardzo (przynajmniej dla mnie) dziwnego formatowania kodu, według mnie tablice są bardzo nieczytelne, długie chainowane funkje rozlewają się w jakiś arbitralny sposób na kilka linijek itp. Jeżeli zmieniy format snapshota żeby odzwierciedlał format planu w aplikacji.

Reply:
2. Jeśli uda się tak zrobić, to będzie super.